File: src/Models/LetsEncryptServer/Account/Requests/PatchHostnameRequest.cs
using MaksIT.Core.Abstractions.Webapi;


namespace MaksIT.Models.LetsEncryptServer.Account.Requests;
public class PatchHostnameRequest : PatchRequestModelBase {
  public string? Hostname { get; set; }

  public bool? IsDisabled { get; set; }
}

File: src/MaksIT.WebUI/src/models/certsUI/account/requests/PatchHostnameRequest.ts
import { boolean, object, Schema, string } from 'zod'
import { PatchRequestModelBase, PatchRequestModelBaseSchema } from '../../../PatchRequestModelBase'

export interface PatchHostnameRequest extends PatchRequestModelBase {
  hostname?: string
  isDisabled?: boolean
}

export const PatchHostnameRequestSchema: Schema<PatchHostnameRequest> = PatchRequestModelBaseSchema.and(
  object({
    hostname: string().optional(),
    isDisabled: boolean().optional()
  })
)
